foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/event/TestEventManager.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package io.servicecomb.foundation.common.event;
+
+import org.junit.Test;
+
+import com.google.common.eventbus.EventBus;
+
+public class TestEventManager {
```

Comment:
this is test for eventBus, not for EventManager
and just call some method, did not verify result.